orttraining/orttraining/training_ops/cuda/optimizer/common.cuh

```diff
@@ -15,11 +15,12 @@ namespace cuda {
 template<typename TLossScale, typename TGradNorm, typename TFinalScale>
 __device__ __forceinline__ TFinalScale _ComputeGradScale(
 const TLossScale* loss_scale,
-const TGradNorm* g_norm) {
+const TGradNorm* g_norm,
+const TFinalScale max_norm) {
```

Comment:
Can we rewrite these piece of code to be more readable ? 

it took us so long to understand this... let's make it less painful for the next reader

Reply:
Just added a comment to explain. I think what we need is adding that comment to this PR for details, not rewriting code.